Dockerfile

```diff
@@ -72,6 +72,8 @@ RUN curl -fsSL http://smithlabresearch.org/downloads/preseq_linux_v2.0.tar.bz2 -
     ln -s /opt/preseq_v2.0/preseq /usr/local/bin/preseq && \
     ln -s /opt/preseq_v2.0/bam2mr /usr/local/bin/bam2mr && \
     rm /opt/preseq_linux_v2.0.tar.bz2
+# Make sure that libgsl.so.0 exists beause PreSeq links to that
+RUN cp /lib/x86_64-linux-gnu/libgsl.so /lib/x86_64-linux-gnu/libgsl.so.0
```

Comment:
Nice! Does this work as an `&& \` softlink like the other above? Would be nice to avoid adding a new command and layer to the docker image.

Also I think it may make more sense to put this command with the initial library installations at the top of the script (keep the comment though, that's helpful).

Bra jobbat!